gradle.properties

```diff
@@ -1,4 +1,4 @@
-org.gradle.jvmargs=-Xmx2g
+org.gradle.jvmargs=-Xmx2g -XX:+UseParallelGC
```

Comment:
Should we replicate the same on our `local.properties` or would this one override the local one?

Reply:
I'm not sure if `org.gradle.jvmargs` inside `local.properties` actually does anything 🤔.

`local.properties` is a Android Gradle Plugin thing, shouldn't affect "non-android-specific" things, unless added by other parties.

Like we did with the GitHub token thingy. We explicitly read from that file. But Gradle doesn't AFAIK.

In any case, this `global.properties` change should affect all developers.